Quadriga/src/main/java/edu/asu/spring/quadriga/service/resolver/impl/ProjectHandleResolverManager.java

```diff
@@ -43,32 +44,54 @@ public boolean saveProjectHandleResolver(IProjectHandleResolver resolver, String
             resolver.setId(id);
             resolver.setUsername(username);
         }
-        
+
         ProjectHandleResolverDTO resolverDto = mapper.mapProjectHandleResolver(resolver);
         resolverDao.saveOrUpdateDTO(resolverDto);
-        
+
         return true;
     }
-    
-    /* (non-Javadoc)
-     * @see edu.asu.spring.quadriga.service.resolver.impl.IProjectHandleResolverManager#getProjectHandleResolver(java.lang.String)
+
+    /*
+     * (non-Javadoc)
+     * 
+     * @see edu.asu.spring.quadriga.service.resolver.impl.
+     * IProjectHandleResolverManager#getProjectHandleResolver(java.lang.String)
      */
     @Override
     public IProjectHandleResolver getProjectHandleResolver(String id) {
         ProjectHandleResolverDTO dto = resolverDao.getDTO(id);
         return mapper.mapProjectHandleResolver(dto);
     }
-    
-    /* (non-Javadoc)
-     * @see edu.asu.spring.quadriga.service.resolver.impl.IProjectHandleResolverManager#getProjectHandleResolvers(java.lang.String)
+
+    /*
+     * (non-Javadoc)
+     * 
+     * @see edu.asu.spring.quadriga.service.resolver.impl.
+     * IProjectHandleResolverManager#getProjectHandleResolvers(java.lang.String)
      */
     @Override
     public List<IProjectHandleResolver> getProjectHandleResolvers(String username) {
         List<ProjectHandleResolverDTO> resolverDtos = resolverDao.getProjectResolversForUser(username);
-        
+
         List<IProjectHandleResolver> resolvers = new ArrayList<IProjectHandleResolver>();
         resolverDtos.forEach(dto -> resolvers.add(mapper.mapProjectHandleResolver(dto)));
-        
+
         return resolvers;
     }
+
+    /**
+     * deletes the resolver object from the database
+     * 
+     * @param resolver
+     *            The project handle resolver to delete.
+     */
+    @Override
+    @Transactional
+    public boolean deleteProjectHandleResolver(IProjectHandleResolver resolver) {
+
+        ProjectHandleResolverDTO resolverDto = mapper.mapProjectHandleResolver(resolver);
+        resolverDao.deleteDTO(resolverDto);
+
+        return true;
```

Comment:
Let's make sure that no project uses this resolver. If there is a project using this resolver, let's throw an exception and tell the user, he can't remove the resolver because it's used.